### PR TITLE
6.3 Validation

### PR DESCRIPTION
**Pull request by yuvirani**
### Description
6.3 Validation

### Check List
- [ ] New functionality includes testing.
  - [ ]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: vendor/github.com/HewlettPackard/oneview-golang/ov/uplink_set.go [hunk 1]
```diff
@@ -42,22 +43,22 @@ type VlanAttributes struct {
 }
 
 type UplinkSet struct {
+	Category                       string               `json:"category,omitempty"`                       //"category":"uplink-sets",
```
Comment (AvijitOS) [inline vendor/github.com/HewlettPackard/oneview-golang/ov/uplink_set.go:46]: As I can see there is no change only rearrangement

File: oneview/resource_hypervisor_cluster_profile.go [hunk 2]
```diff
@@ -442,6 +487,7 @@ func resourceHypervisorClusterProfile() *schema.Resource {
 			"state": {
 				Type:     schema.TypeString,
 				Optional: true,
```
Comment (AvijitOS) [inline oneview/resource_hypervisor_cluster_profile.go:489]: i think it should be set as computed: true no optional

File: oneview/resource_hypervisor_cluster_profile.go [hunk 3]
```diff
@@ -452,11 +498,13 @@ func resourceHypervisorClusterProfile() *schema.Resource {
 			"status": {
 				Type:     schema.TypeString,
 				Optional: true,
+				Computed: true,
```
Comment (AvijitOS) [inline oneview/resource_hypervisor_cluster_profile.go:501]: i think it should be set as computed: true no optional

File: oneview/resource_hypervisor_cluster_profile.go [hunk 4]
```diff
@@ -452,11 +498,13 @@ func resourceHypervisorClusterProfile() *schema.Resource {
...
 			},
 
 			"type": {
 				Type:     schema.TypeString,
 				Optional: true,
+				Computed: true,
```
Comment (AvijitOS) [inline oneview/resource_hypervisor_cluster_profile.go:507]: i think it should be set as computed: true no optional

File: oneview/resource_hypervisor_cluster_profile.go (hunk 2 above)
Comment (yuvirani) [reply oneview/resource_hypervisor_cluster_profile.go:489]: its not a read only field for HCP. I checked in Rest guide.

File: oneview/resource_hypervisor_cluster_profile.go (hunk 4 above)
Comment (yuvirani) [reply oneview/resource_hypervisor_cluster_profile.go:507]: This is read only. I will change it.

File: oneview/resource_hypervisor_cluster_profile.go (hunk 3 above)
Comment (yuvirani) [reply oneview/resource_hypervisor_cluster_profile.go:501]: This also not a read only attribute

File: vendor/github.com/HewlettPackard/oneview-golang/ov/uplink_set.go (hunk 1 above)
Comment (yuvirani) [reply vendor/github.com/HewlettPackard/oneview-golang/ov/uplink_set.go:46]: Yes this is from golang done by Nabhajit...

File: oneview/resource_hypervisor_cluster_profile.go (hunk 3 above)
Comment (AvijitOS) [reply oneview/resource_hypervisor_cluster_profile.go:501]: state, status, type, etag, created, modified these are common properties for any resource calls. By nature those are read only

**Review by nabhajit-ray — APPROVED** [review]
If idempotency check is done  after adding the optional field then LGTM.

File: README.md [hunk 5]
```diff
@@ -206,7 +206,7 @@ This feedback is important for us to deliver a useful product.
 
 [HPE OneView Firmware Management White Paper](http://hpe.com/info/OneView/docs)
 
-Note: Currently this SDK supports OneView API 3000 minimally where we can test OneView API 3000 version with this SDK. No new fields have been added/deleted to support API3000 version. Complete support will be done in next releases.If  API version is not provided then appliance's API version will be used. If API version used is not supported then error will be thrown.
+Note: Currently this SDK supports OneView API 3200 minimally where we can test OneView API 3200 version with this SDK. No new fields have been added/deleted to support API3000 version. Complete support will be done in next releases.If  API version is not provided then appliance's API version will be used. If API version used is not supported then error will be thrown.
```
Comment (AvijitOS) [inline README.md:209]: change this "No new fields have been added/deleted to support API3000 version" to "No new fields have been added/deleted to support API 3200 version"

**Review by AvijitOS — APPROVED** [review]

**Pull request merged**